internal/configs/virtualserver_test.go

```diff
@@ -4552,7 +4554,7 @@ func TestGenerateVirtualServerConfigForVirtualServerRoutesWithDos(t *testing.T)
 			AppProtectDosName:            "my-dos-juice",
 			AppProtectDosAccessLogDst:    "svc.dns.com:123",
 			AppProtectDosPolicyFile:      "",
-			AppProtectDosLogConfFile:     "",
```

Comment:
Is this meant to be removed?

Reply:
fix in commit: 
[Code review fixes](https://github.com/nginxinc/kubernetes-ingress/pull/5824/commits/5c3eeeb2238cc007d0f9f8ee42e3d3ca89617f79)